features/auditing.rst

```diff
@@ -183,4 +183,16 @@ There is no further configuration for AuditEvent logging. If you include it in t
 
 For transactions and batches the audit plugin will create an AuditEvent for the transaction/batch as a whole *and* one for every entry in the transaction/batch.
 
-Firely Server does not allow you to update or delete the AuditEvent resources through the RESTful API so the Audit log cannot be tampered with. You can of course still manipulate these resources directly on the database, for instance to offload a surplus of old AuditEvent resources elsewhere. Please :ref:`vonk-contact` us for details if you want to do this.
\ No newline at end of file
+Firely Server does not allow you to update or delete the AuditEvent resources through the RESTful API so the Audit log cannot be tampered with. You can of course still manipulate these resources directly on the database, for instance to offload a surplus of old AuditEvent resources elsewhere. Please :ref:`vonk-contact` us for details if you want to do this.
+
+You can download examples of AuditEvent resources generated by the server below:
```

Comment:
Downloading the audit events is a good start. For readability I would suggest to create a table comparing the structured log elements with the FHIRPath of the element in the resource.